lib/celluloid.rb

```diff
@@ -478,17 +478,30 @@ def future(meth = nil, *args, &block)
 
 # Configure default systemwide settings
 
-Celluloid.task_class = begin
-  Celluloid.const_get(ENV['CELLULOID_TASK_CLASS'] || fail(TypeError))
-rescue
-  Celluloid::Task::Fibered
-end
 
-Celluloid.group_class = begin
-  Celluloid::Group.const_get(ENV['CELLULOID_GROUP_CLASS'] || fail(TypeError))
-rescue
-  Celluloid::Group::Spawner
-end
+Celluloid.task_class =
+  begin
+    str = ENV['CELLULOID_TASK_CLASS'] || 'Fibered'
+    Kernel.const_get(str)
```

Comment:
The thing is: other people actually shouldn't follow the convention. One example is stubbing - e.g. MySpecs::SerialPool should be acceptable. Forcing the constant to belong to Group is only necessary if you're iterating through the available groups ... which probably doesn't make sense in the context of Celluloid.

I'd say it's actually better to *not* allow people to use that namespace - then, if something ever crashes, you can have Celluloid blame the user if the class isn't in the Celluloid namespace (meaning - it's not a class that's part of Celluloid's "approved" classes).

Here, though, what's important for me is to get an exception if I mistype 'Threaded', because suddenly the tests will be using Fibered ... and they can pass. So I want it to fail if my class doesn't exist (e.g. when you switched the names, my tests started succeeding when they shouldn't).